tokio/src/net/tcp/stream.rs

```diff
@@ -392,7 +384,7 @@ impl TcpStream {
     ///
     /// For more information about this option, see [`set_recv_buffer_size`].
     ///
-    /// [`set_recv_buffer_size`]: #tymethod.set_recv_buffer_size
+    /// [`set_recv_buffer_size`]: #method.set_recv_buffer_size
```

Comment:
We probably should update links to path links instead :) I believe it would be:

```suggestion
    /// [`set_recv_buffer_size`]: TcpStream::set_recv_buffer_size
```